app/models/potential_match.rb

```diff
@@ -104,9 +104,14 @@ def self.generate_in_background(collection_id)
       required_types = settings.required_types.map {|t| t.classify}
 
       # treat each signup as a request signup first
-      collection.signups.find_each do |signup|
+      # because find_each doesn't give us a consistent order, but we don't necessarily
```

Comment:
What ordering problem did you find with find_each? The Rails documentation says: "[Order] is automatically set to ascending on the primary key (“id ASC”) to make the batch ordering work." So I would expect that to do what you want?